frontend/frontend/src/components/Game.vue

```diff
@@ -35,6 +35,11 @@ export default {
 		Timer,
 	},
 	mounted() {
+<<<<<<< HEAD
+		let artist = this.$route.query.artist
+		console.log(artist)
+=======
+>>>>>>> main
```

Comment:
```suggestion
		let artist = this.$route.query.artist
		console.log(artist)
```

Reply:
Hey Mo, you merge in main but you didn't resolve conflicts. you need to make sure that places with <<<<< HEAD and main are sorted to have to correct information saved